poetry/core/masonry/metadata.py

```diff
@@ -52,8 +52,11 @@ def from_package(cls, package: "Package") -> "Metadata":
         meta.version = normalize_version(package.version.text)
         meta.summary = package.description
         if package.readme:
-            with package.readme.open(encoding="utf-8") as f:
-                meta.description = f.read()
+            description = []
+            for readme in package.readme:
+                with readme.open(encoding="utf-8") as f:
+                    description += [f.read()]
```

Comment:
```suggestion
                    description.append(f.read())
```

The allocation of another list is unnecessary/wasteful here -- I think `.append()` is cleaner to read.